### PR TITLE
[WFCORE-4227] Add documentation about the ability of the CLI SSL securi…

### PR DESCRIPTION
**Pull request by MMarus**
…ty commands to obtain a server certificate from Let's Encrypt

WFCORE issue: https://issues.jboss.org/browse/WFCORE-4227
EAP issue: https://issues.jboss.org/browse/EAP7-1100
Analysis document: wildfly/wildfly-proposals#156
WFCORE PR: https://github.com/wildfly/wildfly-core/pull/3618/files

Please do not merge before https://issues.jboss.org/browse/EAP7-1150 is fixed.

Description: Add documentation for the ability of the CLI SSL security commands to obtain a server certificate from Let's Encrypt.

### CODE REVIEW

**Comment by darranl** [comment]
Can we please ensure the commit messages only reference the WFLY Jira issues?  

These messages end up in the commit history in the repo and are the only publicly accessible Jira issues.

File: docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc [hunk 1]
```diff
@@ -566,6 +566,11 @@ Do you confirm y/n :y
 ----
 NB: Once the command is executed, the CLI will reload the server and reconnect to it.
 
+This command can also obtain the certificates from the Let's Encrypt certificate authority
+by use of the --lets-encrypt parameter.
+Besides the mentioned workflow the user will be prompted to specify more information
+such as info about account key store and certificate authority account, in order to obtain the certificate from Let's Encrypt.
```
Comment (jfdenise) [inline docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc]: I would replace with:
Besides the mentioned workflow the user will be prompted to specify more information (eg: account key store,  certificate authority account) to obtain the certificate from Let's Encrypt.
Seems to sound better. Applies also to other place.

Comment (kabir) [reply docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc]: @MMarus Can you fix^^?

Comment (MMarus) [reply docs/src/main/asciidoc/_admin-guide/CLI_Recipes.adoc]: @kabir  fixed :). Sorry I probably overlooked this.

**Pull request merged**